scripts/dockcross-manylinux-build-module-deps.sh

```diff
@@ -74,6 +74,8 @@ for MODULE_INFO in ${ITK_MODULE_PREQ_TOPLEVEL//:/ }; do
 
   echo "Cleaning up module dependency"
   cp ./${MODULE_NAME}/include/* include/
+  find ${MODULE_NAME}/wrapping -name '*.in' -print -exec cp {} wrapping \;
```

Comment:
Is an equivalent thing required for Windows and/or Mac?

Reply:
That sounds good to me.

Reply:
Ok done. I have also added the copy of generated include files to the MacOS script for package deps.